tests/suites/test_suite_pkwrite.data

```diff
@@ -1,47 +1,95 @@
 Public key write check RSA
 depends_on:MBEDTLS_RSA_C:MBEDTLS_BASE64_C
-pk_write_pubkey_check:"data_files/server1.pubkey"
+pk_write_pubkey_check:"data_files/server1.pubkey":0
```

Comment:
Can't comment on the binary files, so commenting here: all data files should have an entry in `tests/data_files/Makefile` - see https://github.com/Mbed-TLS/mbedtls/pull/6838#discussion_r1168341616

Reply:
On another front: we should define an enum or macros in the `.function` file and use them here instead of using 0 and 1 for the last parameter. (The function's signature will still have `int` because our testing generation framework doesn't support more, but the data file will be more readable with say `TEST_PEM` vs `TEST_DER` than with 0 vs 1.

Reply:
> all data files should have an entry in tests/data_files/Makefile

Right, I was tempted to add that (since I already did in the past for similar things), so I went looking for entries generating previous files, but I didn't find them so I was not sure if I could do so or not. Anyway I agree that it is nice to have the commands there at least as sort of documentation, so I will add them ;)

Reply:
Got me, I should have written "all _new_ data files" :) A lot of data files pre-date this `Makefile` and we didn't go back and add them all (so far?).